zellij-server/src/tab.rs

```diff
@@ -2513,6 +2513,31 @@ impl Tab {
             }
         }
     }
+    pub fn scroll_active_terminal_up_half_page(&mut self) {
+        if let Some(active_terminal_id) = self.get_active_terminal_id() {
+            let active_terminal = self
+                .panes
+                .get_mut(&PaneId::Terminal(active_terminal_id))
+                .unwrap();
+            // prevent overflow when row == 0
+            let scroll_columns = (active_terminal.rows().max(1) - 1)/2;
```

Comment:
Should be `scroll_rows`, right?

Reply:
Yes! But to be fair this is also a mistake in the other methods this was copied from. If you'd like to fix this @oromate that would be cool.

Reply:
I can fix this. Do you think it makes sense for me to also correct the variable name in the other method?

Reply:
That would be great, thanks! There are a few of those.